src/util/highlight.js

```diff
@@ -3,13 +3,47 @@
 import hljs from 'highlight.js/lib/highlight';
 
 // Each language needs to be imported like so:
+import bash from 'highlight.js/lib/languages/bash';
+import clojure from 'highlight.js/lib/languages/clojure';
+import cs from 'highlight.js/lib/languages/cs';
+import fortran from 'highlight.js/lib/languages/fortran';
 import go from 'highlight.js/lib/languages/go';
+import haskell from 'highlight.js/lib/languages/haskell';
 import java from 'highlight.js/lib/languages/java';
+import javascript from 'highlight.js/lib/languages/javascript';
+import lisp from 'highlight.js/lib/languages/lisp';
+import lua from 'highlight.js/lib/languages/lua';
+import matlab from 'highlight.js/lib/languages/matlab';
+import perl from 'highlight.js/lib/languages/perl';
+import php from 'highlight.js/lib/languages/php';
 import python from 'highlight.js/lib/languages/python';
+import r from 'highlight.js/lib/languages/r';
+import ruby from 'highlight.js/lib/languages/ruby';
+import scala from 'highlight.js/lib/languages/scala';
+import sql from 'highlight.js/lib/languages/sql';
+import x86asm from 'highlight.js/lib/languages/x86asm';
```

Comment:
We should support an assembly language, this is the most popular/readable. I stand by it.